queries.sql

```diff
@@ -133,4 +133,11 @@ JOIN species specie ON animal.species_id = specie.id
 WHERE vet.name = 'Maisy Smith'
 GROUP BY specie.name
 ORDER BY visits DESC
-LIMIT 1;
\ No newline at end of file
+LIMIT 1;
+
+explain analyze SELECT COUNT(*) FROM visits where animal_id = 4;
+
+explain analyze SELECT * FROM visits where vet_id = 2;
```

Comment:
- [ ]  Great job working on the first query but I think the second and the third query they not show the best for the explain analyze since the excution time is still more that 1 SEC but it should be lower that 1 SEC so you can just make explain analyze more that one time to reach to the perfect execution time so kindly make that to have after making explain analyze the excution time lower than 1 SEC 🔥🙏🏻💯♻✅💪🏻